qa/integration-tests-engine/pom.xml

```diff
@@ -176,6 +176,17 @@
       -->
     </dependency>
 
+    <!--
+      needed for offline shrinkwrap,
+      BOM must be installed before this module is built
+    -->
+    <dependency>
+      <groupId>org.camunda.bpm</groupId>
+      <artifactId>camunda-bom</artifactId>
+      <version>${project.version}</version>
+      <type>pom</type>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Similar to `camunda-engine-rest-core`, shouldn't the BOM already reliably come first in the reactor build since
1. this project's parent is `camunda-qa`
2. `camunda-qa`'s parent is `database-settings`
3. `database-settings`' parent is `parent`
4. `parent` depends on BOM to be build